docs/SPIR-V.rst

```diff
@@ -347,7 +347,65 @@ Optimization
 
 Optimization is also delegated to SPIRV-Tools. Right now there are no difference
 between optimization levels greater than zero; they will all invoke the same
-optimization recipe. This may change in the future.
+optimization recipe. If you want to run a custom optimization recipe, you can do
+so using ``-Oconfig=`` and specifying a comma-separated list of your desired passes.
+
+For example, you can specify: ``-Oconfig=--loop-unroll,--eliminate-dead-variables,--scalar-replacement=300``.
+
+Here is a list of valid passes:
+
+* ``--ccp``
```

Comment:
Sorry, but we should clean this list a little bit. Some of the passes are superseded by others; we should not recommend them. Also we should avoid mentioning `--legalize-hlsl`, which is not supposed to be invoked dedicatedly by the developer, and put `-O` and `-Os` separately.